tutorials/matchmaker101/director/Dockerfile

```diff
@@ -15,8 +15,10 @@
 FROM golang:alpine as go
 WORKDIR /app
 ENV GO111MODULE=on
-
+RUN apk add --no-cache git
 COPY . .
+RUN go mod edit -replace open-match.dev/open-match@v0.0.0-dev=open-match.dev/open-match@latest
```

Comment:
By adding the latest tag you're pulling the most recent versions image (e.g 1.4.0-rc.1). For the main branch we try to avoid this and ask devs to switch to the version branch of their choosing. If there is a larger reason for this to happen, happy to listen to your suggestions/concerns.

Reply:
No reason - is there a version you'd suggest? Or a tag that represents something like "stable"